test/rxd/3d/test_multi_gridding_mix.py

```diff
@@ -0,0 +1,57 @@
+from neuron.units import µm, mM, ms, mV
+import sys
+
+sys.path.append("..")
```

Comment:
Is this necessary?

Reply:
we had trouble running it locally without that